src/modules/fancyzones/FancyZonesModuleInterface/dllmain.cpp

```diff
@@ -158,8 +167,12 @@ class FancyZonesModuleInterface : public PowertoyModuleIface
             Trace::FancyZones::EnableFancyZones(false);
         }
 
-        ResetEvent(m_toggleEditorEvent);
-        CloseHandle(m_toggleEditorEvent);
+        if (m_toggleEditorEvent)
+        {
+            ResetEvent(m_toggleEditorEvent);
+            CloseHandle(m_toggleEditorEvent);
+            m_toggleEditorEvent = nullptr;
```

Comment:
It's a bad place to close m_toggleEditorEvent. It should be closed in the destructor as it is created in the constructor.

Reply:
- Disable/Enable FancyZones 
- Try to use `Launch layout editor` button